packages/standard-minifier-css/plugin/minify-css.js

```diff
@@ -81,13 +115,15 @@ class CssToolsMinifier {
     const { error, postcssConfig } = await loadPostCss();
 
     if (error) {
+      if (verbose) console.log('processFilesForBundle loadPostCss error', error);
```

Comment:
Again, would prefer to use the logging package or at least `console.error` I think would work better here.
https://docs.meteor.com/packages/logging.html

Reply:
Oh, I guess I should have read all of your comments first. I see you're recommending the logging package.

I assumed that the user would need to add the logging package to be able to see anything written with Log. But I will check it out and revert.
It would hurt developer experience if they enable `--verbose` mode, but can't see anything if they haven't installed the logging package.